framework/src/modules/chain/components/storage/entities/round_delegates.js

```diff
@@ -71,6 +71,10 @@ class RoundDelegates extends BaseEntity {
 		this.SQLs = this.loadSQLFiles('dpos', sqlFiles, this.sqlDirectory);
 	}
 
+	/**
+	 * @returns {string[]} delegatePublicKeys
+	 */
+
```

Comment:
Minor thingy:
- Remove empty line between description and function declaration
- This code has been there, but it is very hard to know for someone reading the code what `result[0]` means? Please add some clarification if possible.